python/LAMBDA_CONCURRENCY_CHECK/parameters.json

```diff
@@ -0,0 +1,12 @@
+{
+  "Version": "1.0",
+  "Parameters": {
+    "RuleName": "LAMBDA_CONCURRENCY_CHECK",
+    "SourceRuntime": "python3.6",
+    "CodeKey": "LAMBDA_CONCURRENCY_CHECK.zip",
+    "InputParameters": "{}",
+    "OptionalParameters": "{}",
```

Comment:
Add the parameters as keys with an empty value.

Reply:
Done!